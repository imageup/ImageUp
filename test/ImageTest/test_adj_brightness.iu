function main() -> int
{
        image: a = read("./images/face1.jpg");
        image: output;
		output = adjust_brightness(a, 20.0);

	    save("test.out.jpg", output);

        return 0;
}
