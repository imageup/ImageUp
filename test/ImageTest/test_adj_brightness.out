Before: 233.000 239.000 238.000
After: 253.000 255.000 255.000
